### PR TITLE
Added possibility to disable payment methods

### PR DESCRIPTION
**Pull request by rogergros**
- Payment methods can be disabled via config file
- Wizard bundle now depends on payment bridge
- Wizard checks only the payment methods enabled

### CODE REVIEW

File: src/Elcodi/Plugin/StoreSetupWizardBundle/ElcodiStoreSetupWizardBundle.php [hunk 1]
```diff
@@ -50,4 +51,16 @@ public function registerCommands(Application $application)
...
+
+    /**
+     * Create instance of current bundle, and return dependent bundle namespaces
+     *
+     * @return array Bundle instances
+     */
+    public static function getBundleDependencies()
+    {
+        return [
+            '\Elcodi\Common\PaymentBridgeBundle\PaymentBridgeBundle',
```
Comment (cordoval) [inline src/Elcodi/Plugin/StoreSetupWizardBundle/ElcodiStoreSetupWizardBundle.php]: can we just do without the \?

Comment (mmoreram) [inline src/Elcodi/Plugin/StoreSetupWizardBundle/ElcodiStoreSetupWizardBundle.php]: Yes. @rogergros Please remove the first `\`.

**Comment by cordoval** [comment]
bump @mmoreram resolved in #720

**Comment by rogergros** [comment]
Solved, I've also fixed some aesthetic problems (Fixer).

**Comment by cordoval** [comment]
:+1:

File: src/Elcodi/Plugin/StoreSetupWizardBundle/Resources/config/services.yml [hunk 2]
```diff
@@ -9,6 +9,7 @@ services:
             configuration_manager: @elcodi.manager.configuration
             product_repository: @elcodi.repository.product
             shipping_range_repository: @elcodi.repository.shipping_range
+            enabled_methods: %store.payment_bridge.enabled_methods%
```
Comment (mmoreram) [inline src/Elcodi/Plugin/StoreSetupWizardBundle/Resources/config/services.yml]: Hmmm, mi no like this. Please, rethink this element, as you cannot use any Store parameter

Comment (mmoreram) [inline src/Elcodi/Plugin/StoreSetupWizardBundle/Resources/config/services.yml]: The way of solving this is adding this configuration as a plugin configuration element.

Comment (mmoreram) [inline src/Elcodi/Plugin/StoreSetupWizardBundle/Resources/config/services.yml]: @rogergros

**Pull request merged**